diff -urN gtubeclock-0.9.1.orig/gtubeclock.desktop gtubeclock-0.9.1/gtubeclock.desktop
--- gtubeclock-0.9.1.orig/gtubeclock.desktop	2004-11-30 22:34:11.656827240 +0100
+++ gtubeclock-0.9.1/gtubeclock.desktop	2004-11-30 22:36:15.328026360 +0100
@@ -1,7 +1,10 @@
 [Desktop Entry]
 Name=gtubeclock
 Comment=Nixie tube clock
+Comment[pl]=Zegar z wyświetlaczem lampowym
 Exec=gtubeclock
-Terminal=0
+Terminal=false
 Type=Application
-Categories=GNOME;Application;Utility;
+Categories=GTK;GNOME;Utility;Clock;
+Encoding=UTF-8
+# vi: encoding=utf-8
